examples/blog/src/styles/global.css

```diff
@@ -46,6 +46,7 @@ table {
 }
 img {
 	max-width: 100%;
+	height: auto;
```

Comment:
Wont this trigger a flash of unsettled content? It would be great to use something like “fill” or “cover” instead on the hero images with an explicit height

Reply:
Here's `height: auto`

https://user-images.githubusercontent.com/7118177/184648691-7d613b61-717b-4935-a7c2-2e868883392d.mp4

vs `object-fit: cover`

https://user-images.githubusercontent.com/7118177/184648744-2cbdb204-a601-4d78-bddf-e730a933c9e6.mp4

Reply:
Also I definitely don't get FOUC as the image loads in, because `width` and `height` are both specified so the aspect ratio is automatically calculated by the browser.

Reply:
+1! I don't see another way to auto-scale the height other than this